### PR TITLE
Dialign fix, again

### PR DESCRIPTION
**Pull request by jfallmann**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

removing pre and post scripts

### CODE REVIEW

**Comment by jfallmann** [comment]
@bgruening Sorry, letzter merge hat das problem gar nicht gefixt, ging wohl nur bei mir lokal

File: recipes/dialign2/patchdialign.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
---- /homes/biertruck/fall/Downloads/dialign_package/src/dialign.c	2005-02-21 17:32:55.000000000 +0100
-+++ /homes/biertruck/fall/Downloads/dialign_package/src/fixed_dialign.c	2018-07-26 16:46:41.962909610 +0200
```
Comment (bgruening) [inline recipes/dialign2/patchdialign.txt]: should this file now be removed?

File: recipes/dialign2/post-link.sh [hunk 2]
```diff
@@ -1,2 +0,0 @@
-echo "In order to use dialign2 you have make sure that the environment variable DIALIGN2_DIR is set to $PREFIX/share/dialign2; will do this not via `export DIALIGN2_DIR=$PREFIX/share/dialign2`"
```
Comment (bgruening) [inline recipes/dialign2/post-link.sh]: This is not needed anymore?

File: recipes/dialign2/patchdialign.txt (hunk 1 above)
Comment (jfallmann) [reply recipes/dialign2/patchdialign.txt]: yes, the patch is now in the subdir patches

File: recipes/dialign2/post-link.sh (hunk 2 above)
Comment (jfallmann) [reply recipes/dialign2/post-link.sh]: this is taken care of with the patch and during the build, no need for pre and post scripts anymore

Comment (jfallmann) [reply recipes/dialign2/post-link.sh]: thx!

**Pull request merged**